src/ts/components/subtitleoverlay.ts

```diff
@@ -73,6 +137,193 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     // Init
     subtitleClearHandler();
   }
+
+  /**
+   * updateSubtitleOverlayConfig updates style for the subtitle overlay based on init config
+   **/
+  private updateSubtitleOverlayFromUIconfig(subtitlesConfig: UISubtitleConfig): void {
+    if (subtitlesConfig == null) {
+      return;
+    }
+    if (subtitlesConfig.backgroundColor != null) {
+      this.backgroundColor = ColorUtils.colorFromCss(subtitlesConfig.backgroundColor);
+    }
+    if (subtitlesConfig.fontColor != null) {
+      this.fontColor = ColorUtils.colorFromCss(subtitlesConfig.fontColor);
+    }
+    if (subtitlesConfig.windowColor != null) {
+      this.windowColor = ColorUtils.colorFromCss(subtitlesConfig.windowColor);
+    }
+    this.fontFamily = subtitlesConfig.fontFamily;
```

Comment:
Apply all of these properties *conditionally* if set (`if (subtitlesConfig.fontFamily) { this.fontFamily = subtitlesConfig.fontFamily; }`. Currently, when setting just one property of the `UISubtitleConfig`, other unset properties would be `undefined` and overwrite the local config with `undefined`.